examples/plotting/custom_tooltip.py

```diff
@@ -1,3 +1,15 @@
+''' A plot of periodic table elements using a Periodic table dataset.
+This example demonstrates the use of custom css tooltips when creating plots.
+The chart shows correlation between atomic mass and density of gssperiodic table elements.
```

Comment:
typo / regularizing line lengths
```suggestion
The chart shows correlation between atomic mass and density of periodic table
elements.
```